libmscore/accidental.h

```diff
@@ -114,6 +114,7 @@ class Accidental final : public Element {
       virtual QString propertyUserValue(Pid) const override;
 
       static AccidentalVal subtype2value(AccidentalType);             // return effective pitch offset
+      static SymId subtype2symbol(AccidentalType);
```

Comment:
or too much here?

Reply:
Could be inlined, I guess

Reply:
It is definitely a static function. And I used the surrounding code as an example in both the declaration and the definition.

Reply:
It is static, it can't get used outside of accidental.cpp. But you do use it outside, so it can't be static. Those other seem wrong then too...